lisa/analysis/tasks.py

```diff
@@ -46,37 +47,38 @@ def __or__(self, other):
             int(self) | int(other),
             char=(self.char + other.char))
 
+
 class TaskState(StateInt, Enum):
     """
     Represents the task state as visible in sched_switch
 
     * Values are extracted from include/linux/sched.h
     * Chars are extracted from fs/proc/array.c:get_task_state()
     """
-    #pylint-suppress: bad-whitespace
-    TASK_RUNNING         = 0x0000, "R", "Running"
-    TASK_INTERRUPTIBLE   = 0x0001, "S", "Sleeping"
+    # pylint-suppress: bad-whitespace
```

Comment:
indeed, it would be better to be able to preserve that, but I don't think we can disable that linter without loosing a lot of actually useful cleanup :/